src/apps/popularFeatured.js

```diff
@@ -28,20 +30,18 @@ const popularFeaturedAppsQuery = `
                             JOIN app_category_group acg ON w.root_category_id = acg.parent_category_id
                             JOIN app_category_app aca ON acg.child_category_id = aca.app_category_id
                    WHERE authenticated_user.username = $1
-                     AND acg.child_index = $2
+                     AND acg.child_index = $3
                      AND aca.app_id = a.id
                )       AS is_favorite,
            true        AS is_public
-    FROM apps a
+    FROM app_listing a
```

Comment:
I definitely worry about swapping to the listing view when we don't need a lot of the stuff it includes/joins on. Would definitely like to see some before/after query plans on this if it's possible to do that, and we can see if it works better to explicitly join only a few tables we actually need. I figure performance here might be especially important since this is for the dashboard.

Reply:
I do remember these endpoints getting a little slower, so I wasn't concerned before, but now I don't remember by how much, so I'll run some more timing tests soon.

I wonder if it would help to create a new `app_latest_versions` view that has all the top-level info queries like this need, and then the `app_listing` view can join on this new view to add extra details like rating info, task counts, step counts, and the like.